Rustix 0.38.22 breaks my build

I'm trying to compile [this app](https://github.com/bihealth/mehari) on branch `main`/`973ab7b`. The program compiled on CI 10h ago but locally I get:

```
error[E0432]: unresolved import `crate::backend::vdso_wrappers`
  --> /home/holtgrem_c/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rustix-0.38.22/src/backend/linux_raw/process/syscalls.rs:46:32
   |
46 | pub(crate) use crate::backend::vdso_wrappers::sched_getcpu;
   |                                ^^^^^^^^^^^^^ could not find `vdso_wrappers` in `backend`
```

Removing `target/`, `~/.cargo` and `~/.rustup` and reinstalling did not help.

The following in `Cargo.toml` fixes the issue:

```
rustix = "=0.38.21"  # pin
```

I'm on `Ubuntu 22.04.3 LTS`

```
uname -a
Linux $HOSTNAME 6.2.0-36-generic #37~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Mon Oct  9 15:34:04 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
```

--- a/src/backend/linux_raw/mod.rs
+++ b/src/backend/linux_raw/mod.rs
@@ -18,9 +18,9 @@
 mod arch;
 mod conv;
 mod reg;
-#[cfg(any(feature = "time", target_arch = "x86"))]
+#[cfg(any(feature = "time", feature = "process", target_arch = "x86"))]
 mod vdso;
-#[cfg(any(feature = "time", target_arch = "x86"))]
+#[cfg(any(feature = "time", feature = "process", target_arch = "x86"))]
 mod vdso_wrappers;
 
 #[cfg(feature = "event")]

--- a/src/backend/linux_raw/vdso_wrappers.rs
+++ b/src/backend/linux_raw/vdso_wrappers.rs
@@ -29,7 +29,7 @@ use core::sync::atomic::Ordering::Relaxed;
 #[cfg(target_pointer_width = "32")]
 #[cfg(feature = "time")]
 use linux_raw_sys::general::timespec as __kernel_old_timespec;
-#[cfg(feature = "time")]
+#[cfg(any(feature = "time", feature = "process"))]
 use {
     super::c,
     super::conv::{c_int, ret},

